tests/integration/targets/setup_pkg_mgr/tasks/main.yml

```diff
@@ -1,10 +1,12 @@
 ---
 - set_fact:
     pkg_mgr: community.general.pkgng
+    ansible_pkg_mgr: community.general.pkgng
```

Comment:
I wonder if it's possible to run setup module for `ansible_pgk_mgr` discovery like here: https://github.com/ansible/ansible/blob/308723c3ca1122363e419070f1fa1d76ff5611a9/lib/ansible/plugins/action/package.py#L54

Reply:
No (besides that that's already done). The problem is that `setup` returns the wrong name, which currently (until ansible/ansible#67684 is merged) does not work. The action plugin `package` will try to find the module `pkgng` (resp. `zypper`), but the modules are called `community.general.pkgng` (resp. `community.general.zypper`) now.

With the routing PR in ansible/ansible, the short name will again work (and this PR can be reverted). But until then, this is needed for make CI work for FreeBSD and OpenSuSE.

Reply:
clear, thanks